src/azure-cli/azure/cli/command_modules/servicebus/tests/latest/test_servicebus_migration_commands.py

```diff
@@ -0,0 +1,169 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# AZURE CLI SERVICEBUS - CRUD TEST DEFINITIONS
+
+import time
+from azure.cli.testsdk import (ScenarioTest, ResourceGroupPreparer, live_only)
+from knack.util import CLIError
+
+
+# pylint: disable=line-too-long
+# pylint: disable=too-many-lines
+
+
+class SBNSMigrationCRUDScenarioTest(ScenarioTest):
+    from azure_devtools.scenario_tests import AllowLargeResponse
+
+    # Test playback fails and the live-only flag will be removed once it is addressed
+    @live_only()
```

Comment:
May I know the reason for test failures?

Reply:
in this test are migrating a standard namespace to premium namespace and the entities within it.  we are investigating why the play back is failing in CI.